app/models/test_passage.rb

```diff
@@ -0,0 +1,66 @@
+class TestPassage < ApplicationRecord
+  SUCCESS_PERCENT = 85
+
+  belongs_to :user
+  belongs_to :test
+  belongs_to :current_question, class_name: 'Question', optional: true
+
+  before_validation :set_first_question, on: %i[create update]
```

Comment:
название не очень удачное )
может так?
```suggestion
  before_validation :set_next_question, on: %i[create update]
```